client/marklogicClient.ts

```diff
@@ -442,7 +442,7 @@ export async function filterJsServerByConnectedStatus(
 
     choices.forEach((choice) => {
         // TODO - manageBasePath
```

Comment:
Do you still need this TODO here?

Reply:
Yes. I'll take care of this in a PR later today.